CRM/Core/Page/AJAX.php

```diff
@@ -108,21 +108,20 @@ public static function setIsQuickConfig() {
     }
     $priceSetId = CRM_Price_BAO_PriceSet::getFor($context, $id, NULL);
     if ($priceSetId) {
-      $result = CRM_Price_BAO_PriceSet::setIsQuickConfig($priceSetId, 0);
+      $sql = "UPDATE
+       civicrm_price_set cps
+       INNER JOIN civicrm_price_set_entity cpse ON cps.id = cpse.price_set_id
+       INNER JOIN {$context} ce ON cpse.entity_id = ce.id
```

Comment:
we cannot use %1 here as the eventual query is translated as ```INNER JOIN 'civicrm_event'```, eventually causes DB error.